PSR standard Composer Package

PabloJoan opened the pull request:
Usable library without having to change the core code base. Installable via Composer

Commit:
#13792 Porting Feature Library to PSR-4 Composer package

Commit:
#13792 Porting Feature Library to PSR-4 Composer package

Commit:
Update .gitignore

Commit:
Update .travis.yml

Commit:
Update composer.json

Commit:
Update phpunit.xml

Commit:
Merge pull request #1 from cafemedia/13792

* #13792 Porting Feature Library to PSR-4 Composer package

* track ticket: http://trac.clubmom.local/trac/cafemom/ticket/13792

Commit:
Added PSR-3 Support

Commit:
Merge pull request #2 from cafemedia/13792

Added PSR-3 Support

Commit:
Change IsAdmin logic

remove Logic from isAdmin and accept Boolean from constructor instead

Commit:
append feature to config errors

log bad feature along with message

Commit:
remove unused parameter

logger does not execute tracking logic

Commit:
typo in comment

Commit:
isAdmin bug in world constructor

Commit:
replacing unused adminIds variables with new isAdmin implementation

Commit:
generalize function name

Commit:
fix calling static properties dynamically

Commit:
1.0.0 stable version with unit tests

Commit:
Delete .travis.yml

Commit:
Update .gitignore

Commit:
Delete composer.lock

Commit:
Update Config.php

Commit:
include build.xml for jenkins integration

Commit:
Update README.md

Commit:
test coverage

Commit:
Merge branch 'master' of github.com:cafemedia/feature

staabm commented:
this lib seems to be no longer maintained. hopefully you dont waste your own time.

Commit:
disabled logging

Commit:
2.0 (#3)

Remove all things static.
added support for 'excluded_from' and 'start', 'end' feature configurations.
remove 'on', 'off' magic that complicates code and replace it with standard [enabled => 100], [enabled => 0]

* adding support for start - end configurations. also exclude from regions support

* forgot to add files (#1)

* fix start - end feature bug

* addin automated testing

* Update README.md

* Delete GENERALIZING.md

* Update README.md

* Update README.md

PabloJoan commented:
@staabm I'm not :)

PHP has no good feature flaging alternative. I will be maintaining this library under https://github.com/pablojoan/feature

composer require pablojoan/feature

100% code test coverage and psr4 compliant

Commit:
adding Jenkins file

Commit:
update jenkinsfile

Commit:
Update Jenkinsfile

Commit:
Update Jenkinsfile

Commit:
Update Jenkinsfile

Commit:
Update Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile

Commit:
Updated Jenkinsfile
